lib/option.rb

```diff
@@ -51,11 +51,11 @@ def self.postgres_locations
     VmSize.new("standard-gpu-#{_1}", "standard-gpu", _1, (_1 * 5.34).to_i, (_1 / 2) * 60, _1 * 60, (_1 / 2) * 60, false, true)
   }).freeze
 
-  PostgresSize = Struct.new(:name, :vm_size, :family, :vcpu, :memory, :storage_size_gib) do
+  PostgresSize = Struct.new(:name, :vm_size, :family, :vcpu, :memory, :min_storage_size_gib, :max_storage_size_gib, :storage_size_step_gib) do
```

Comment:
Instead of min, max and step, should we simply list the available sizes in an array? I think it's cleaner